neutron_vpnaas/db/vpn/ovn_vpn.py

```diff
@@ -0,0 +1,207 @@
+#    (c) Copyright 2016 IBM Corporation, All Rights Reserved.

```

Comment:
Hi Bonnie,
I just sync the code from the master, seems there are some new validation codes, not sure if it has effects to your code